dash.js

```diff
@@ -78,6 +78,28 @@ const MyDashActor = new Lang.Class({
         this.actor.connect('allocate', Lang.bind(this, this._allocate));
 
         this.actor._delegate = this;
+
+        let alignment = 0.25;
+        if (this._isHorizontal)
+            alignment = 0.5;
+        // Popup menu
+        this.menu = new PopupMenu.PopupMenu(this.actor, alignment, this._position);
+        this.menu.actor.hide();
+        this.menu.actor.add_style_class_name('panel-menu');
```

Comment:
Ah, thanks for spotting it! I don't think it's strictly needed, I might have added when trying different styles. Let me remove that, check and resubmit.